echopype/echodata/combine.py

```diff
@@ -65,20 +69,25 @@ def check_zarr_path(
         save_path=zarr_path,
     )
 
+    # convert created validated_path to a Path object
+    if zarr_path is None:
+        validated_path = Path(validated_path)
```

Comment:
This will break a path that's `s3://my-bucket/data.zarr`.

Reply:
`validate_output_path` already deals with the complexity of `Path`. The downstream should just take the regular string of `validated_path`.

Reply:
I see how my implementation has issues. Thank you for pointing this out. In your opinion, should I make it so that `check_zarr_path` always outputs a string? From your comments, this seems like the best solution.

Reply:
Yepp, `check_zarr_path` should always output a string. Thanks for working on this!

Reply:
Sounds good! I will go ahead and implement this.